ext/pcntl/tests/pcntl_getpriority_error.phpt

```diff
@@ -13,6 +13,14 @@ if (PHP_OS == "Darwin") {
     die("skip This test is not for Darwin");
 }
 
+$envUser = getenv("USER") ?: getenv("USERNAME");
```

Comment:
would it be savy to put those in a common file and include it instead ?

Reply:
Hello @devnexen , I will put that logic in a file and include it in all tests. Thanks for the feedback

Reply:
@devnexen , did I understood it correctly? let me know, I am open to adjust as much as needed

Reply:
you did.